Zend/Optimizer/sccp.c

```diff
@@ -836,6 +837,7 @@ static bool can_ct_eval_func_call(zend_string *name, uint32_t num_args, zval **a
 		|| zend_string_equals_literal(name, "str_split")
 		|| zend_string_equals_literal(name, "str_starts_with")
 		|| zend_string_equals_literal(name, "strpos")
+		|| zend_string_equals_literal(name, "strstr")
```

Comment:
strtolower etc ashould be added as well

Reply:
According to Nikita, it is locale dependant and can't be included here.